src/test/java/dk/cngroup/lentils/service/ProgressServiceIntegrationTest.java

```diff
@@ -9,12 +9,12 @@
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.transaction.annotation.Transactional;
 
 import static org.junit.Assert.assertEquals;
 
-@RunWith(SpringRunner.class)
+@RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
Pokud by to šlo tak bych asi taky nechal SpringRunner jak jsme se bavili předtím :-)